### DISCUSSION TITLE
projection into the fundamental region

### DISCUSSION THREAD

**Discussion by Saoussen-ouhiba** [mtex-toolbox/mtex#720] [Ask Anything] [answered]
Hello,

I tried to plot some misorientation angle/axis pairs in the fundamental region (see figure1). By comparing figure 1 with figure 2 which is obtained by considering the upper hemisphere with antipodal symmetry, I think that not all the points are projected onto the fundamental region. But I am not sure because I am not very familiar with spherical projections. So could you please confirm that to me? and if so, could you please help me to solve the problem? The crystal symmetry is m-3m. 
Thank you a lot in advance,
Saoussen
![figure1](https://user-images.githubusercontent.com/73701277/105560391-e6085680-5d13-11eb-8d0f-7ed45cdab79e.jpg)
![figure2](https://user-images.githubusercontent.com/73701277/105560452-0801d900-5d14-11eb-9ced-c96fec30c197.jpg)

**Comment by ralfHielscher** [comment]
Most probably you did something wrong. But without any code its hard to say what.

Ralf.

**Reply by Saoussen-ouhiba** [reply]
Thank you for your reply. My script is attached below. It would be great if you could verify it for me and let me know what's wrong in it. Thank you again very much.
```
cs = crystalSymmetry('m-3m');
ss = specimenSymmetry('mmm');

% definition of CSTM component
CSTMa = orientation('euler',74*degree,23*degree,37*degree,'ZXZ',cs);
CSTMb = orientation('euler',254*degree,23*degree,37*degree,'ZXZ',cs);
CSTMc = orientation('euler',106*degree,157*degree,37*degree,'ZXZ',cs);
CSTMd = orientation('euler',286*degree,157*degree,37*degree,'ZXZ',cs);

% definition of goss component
goss = orientation.goss(cs,ss);

% definition of cube component
cube = orientation.cube(cs,ss);

% definition of copper component
copper = orientation.copper(cs,ss);
copper2 = orientation.copper2(cs,ss);

% definition of brass component
brass = orientation.brass(cs,ss);
brass2 = orientation.brass2(cs,ss);

% definition of S component
S = orientation.SR(cs,ss);
S2 = orientation.SR2(cs,ss);
S3 = orientation.SR3(cs,ss);
S4 = orientation.SR4(cs,ss);

% calculate misorientation angle/axis between cube component and other
% texture components
RexComponent= cube;
DeformedComponents=[CSTMa,CSTMb,CSTMc,CSTMd, copper, copper2, goss,brass, brass2, S, S2, S3, S4];

for j = 1:13
mori(1,j) = inv(RexComponent(1)) * DeformedComponents(j);
end

Angles = mori.angle./degree;

Axes = mori.axis;

figure
plot(Axes,Angles,'MarkerSize',10,'MarkerEdgeColor','k','antipodal','fundamentalRegion')
mtexColorbar
caxis([10 60]) 
```

**Reply by kilir** [reply]
Hi,
you were actually missing `project2FundamentalRegion`. See here https://mtex-toolbox.github.io/Miller.project2FundamentalRegion.html
Cheers,
Rüdiger

**Reply by Saoussen-ouhiba** [reply]
Hi Rüdiger,

Thank you so much for your reply. Unfortunately, even with project2FundamentalRegion, I still get the same result (missing points on the stereographic triangle). Script to generate the figure is below.
```
figure
plot(Axes.project2FundamentalRegion,Angles,'MarkerSize',10,'MarkerEdgeColor','k','fundamentalRegion','antipodal')
mtexColorbar
caxis([10 60])
``` 
Thank you a lot and sorry for bothering you again,
Saoussen

**Reply by ralfHielscher** [reply]
Just for completeness, the above code did not work as expected, as `project2FundamentalRegion` was not aware of the option `antipodal` you included into the plot command.
```matlab
plot(Axes.project2FundamentalRegion('antipodal'),Angles,'fundamentalRegion','antipodal')
```
shall work as well.
Ralf.

**Comment by ralfHielscher** [comment, marked as answer]
may be you try
```matlab
plot(Axes,Angles,'MarkerSize',10,'MarkerEdgeColor','k','fundamentalRegion','antipodal','symmetrised')
```
Ralf.

**Reply by Saoussen-ouhiba** [reply]
yes, it is working now. Thanks a lot,
Saoussen
